infobar-with-user-attribute-and-cta/client.js

```diff
@@ -46,8 +48,8 @@
         Evergage.cashDom("body").css({ "margin-top": "0", "margin-bottom": "0" });
     }
 
-    function control() {
-
+    function control(context) {
+        return !!context.contentZone;
```

Comment:
Control function can only return a Promise or void, and can't return a boolean. Just do what you did in apply()
```suggestion
        if (!context.contentZone) return;
```

Reply:
https://github.com/evergage/evergage-global-templates/pull/28#discussion_r551612079